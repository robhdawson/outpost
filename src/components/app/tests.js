import React from 'react';
import { mount, shallow } from 'enzyme';

import App from './index';

it('Renders without crashing', () => {
  mount(<App />);
});

it('Renders correctly', () => {
    const app = shallow(<App />)
    expect(app).toMatchSnapshot();
});
