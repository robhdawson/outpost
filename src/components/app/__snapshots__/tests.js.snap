// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="App"
>
    <div
        className="App-header"
    >
        <h2>
            Welcome to React
        </h2>
    </div>
    <p
        className="App-intro"
    >
        To get started, edit 
        <code>
            src/App.js
        </code>
         and save to reload.
    </p>
</div>,
  "nodes": Array [
    <div
      className="App"
>
      <div
            className="App-header"
      >
            <h2>
                  Welcome to React
            </h2>
      </div>
      <p
            className="App-intro"
      >
            To get started, edit 
            <code>
                  src/App.js
            </code>
             and save to reload.
      </p>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <App />,
      "_debugID": 10,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": App {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="App"
>
          <div
                    className="App-header"
          >
                    <h2>
                              Welcome to React
                    </h2>
          </div>
          <p
                    className="App-intro"
          >
                    To get started, edit 
                    <code>
                              src/App.js
                    </code>
                     and save to reload.
          </p>
</div>,
        "_debugID": 11,
        "_renderedOutput": <div
          className="App"
>
          <div
                    className="App-header"
          >
                    <h2>
                              Welcome to React
                    </h2>
          </div>
          <p
                    className="App-intro"
          >
                    To get started, edit 
                    <code>
                              src/App.js
                    </code>
                     and save to reload.
          </p>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <App />,
}
`;
